MQ2DanNet/MQ2DanNet.cpp

```diff
@@ -2571,6 +2571,16 @@ std::string unescape_string(const std::string& input) {
     return std::string(szOut);
 }
 
+// The name sent to be checked against mq2dannet must be all lower case otherwise this will always return false.
+PLUGIN_API bool MQ2DanNetAreTheyConnected(char* szName)
```

Comment:
use the MQ2DANNET_NODE_API and put this in the Node singleton class. Also, this name is far too long and the standard for dannet is snake case, something like `peer_connected` or `is_connected` would be more appropriate. This argument should be a `const std::string&` for safety and mutability reasons.

Reply:
It's your sandbox.  I'll make those changes tomorrow.